osmnx/utils.py

```diff
@@ -198,6 +198,91 @@ def euclidean_dist_vec(y1, x1, y2, x2):
     return distance
 
 
+def get_unique_nodes_ordered_from_way(way_edges_df):
```

Comment:
Does this function get tested during the CI tests? It looks like test coverage drops significantly in this PR.

Reply:
done